core/server/master/src/main/java/alluxio/master/metastore/ReadOnlyInodeStore.java

```diff
@@ -28,29 +28,114 @@
  * Read-only access to the inode store.
  */
 public interface ReadOnlyInodeStore extends Closeable {
+  /**
+   * Options of reading from the inode store.
+   */
+  class ReadOption {
+    private static final ReadOption DEFAULT = new ReadOption(false);
+
+    private final boolean mSkipCachingAndEviction;
```

Comment:
Is there any way to skip the cache but NOT skip eviction? Maybe just `skipCache` is a good enough name?